### ISSUE TITLE
HELP ] What do Sleek users do for Android client?

### ISSUE THREAD

**Issue by hoshi411** [ransome1/sleek#454]
I know that there is no Sleek for Android and so my question is,
What do you people do for syncing the todo.txt files and viewing and editing them in Android phones?

todo.txt apk only allows sync with Dropbox and it only allows to sync a single .txt file at a time.
If you want to use Nexcloud or ANYTHING BUT dropbox/Google, youre out of luck.
Plus it only allows you to sync a single txt file.  If you manage your todo.txt using multiple files, you're in the lake.

So,

What do you guys do?!?!

There must be some solution I am missing.

Sleek is an amazing app but it's biggest drawback to using it is that there is not satisfactory solution for Android. And if there is no Android solution (satisfactory) for todo.txt then no one can use todo.txt as a solution in the first place and furthermore , no one will then be able to adopt Sleek.

Ain't nobody gonna be like , "wish I could manage all my GTD anywhere except my phone.  OH, I dearly wish there was a desktop only solution..... wait. there is?  Sleek?  Oh this is perfect.  Now I can be prohibited from doing any GTD on my phone.  Oh boy.  Exactly what I was looking for!!!"

We need a legitimate  android solution.

**Comment by ransome1** [comment]
@hoshi411 have you checked Simpletask? https://github.com/mpcjanssen/simpletask-android

**Comment by ulville** [comment]
On my laptops and desktops I use VS Code for markdown note taking and Sleek for todo.txt. On Android I use [Markor](https://github.com/gsantner/markor) for both my markdown note taking and todo.txt needs. To sync my notes and todo.txt I use [syncthing](https://syncthing.net/). **But** while Markor is great for a lot of things, its todo.txt presentation is less then ideal. Especially for an Android app. It has tools for sorting and adding attributes to tasks but UI/UXvise it's basically a plain text editor with some syntax highlighting. It doesn't feel comfortable with the touch screen. Simpletask is not bad. But it doesn't highlight `+project` an `@context` parts and looks a bit blend and overall looks outdated. For now I'm making shift with available tools but I would like to see an Android app which is as "sleek" as Sleek

**Comment by ulville** [comment]
PS: And AFAIK none of them has notification support or any other integration with the Android system. Which is something Sleek does on supported platforms

**Comment by Mirodin** [comment]
I feel an urge to defend Simpletask because imho it is amazing and by far the best todo.txt compliant task manager app for Android out there:

1. It does highlight project, context and priority. When creating a new task one can either type a new one or use a list of projects and tasks that are already in the todo.txt file.
3. It does do notifications: It does even do them for due **and** threshold tasks (or one of them, or none). For this it creates a new calendar in the default calendar app so other apps could (theoretically) read that as well. One can choose a time those notifications should trigger. I have mine set to 8:00, because that is when I reach the office.
4. It has built in Lua so can be extended (though I only scratched the surface on that).
5. It provides widgets and theming (maybe that might fix your outdated impression?). On my homescreen I have a translucent widget over the full screen size so my tasks are always there.
6. Every widget can sort/filter tasks on its own. I have my main widget sort by priority and two other widgets on the second homescreen. One lists only project that passed their threshold and have a due date, the other shows a list of all overdue tasks.

I also sync my todo.txt file with Syncthing. By far the best method to not rely on the cloud.

P.S.: Sorry for hijacking the thread :)

**Comment by ulville** [comment]
@Mirodin I tried Simpletask again after writing my last comment. I realized that it has notification support indeed and its widget is very handy. I'm using it for a few days now and started liking it more and more. 
By highlighting projects and contexts I mean showing them in a different color or maybe in a colored background like sleek does. Simpletask shows them just a little bit lighter than the body of the task but It's not very distinguishable.
Theming helps but I wish it also had an option to follow the system theme (dark or light). I may create an feature request about it on its github page (if anyone hasn't done it yet)
